lib/src/Puzzle.dart

```diff
@@ -103,8 +147,11 @@ class Puzzle {
   /// I can only make these comments so interesting and no more :l
   void startStopwatch() => _stopwatch.start();
   void stopStopwatch() => _stopwatch.stop();
-  Duration getTimeElapsed() => _stopwatch.elapsed;
+  /// Add the time elapsed in case the game is being reloaded from map/storage
+  Duration getTimeElapsed() => Duration(seconds: _timeElapsedInSeconds) + _stopwatch.elapsed;
```

Comment:
If [Line 47](https://github.com/JoseGeorges8/sudoku-api/blob/95b4dff6533dbff802540617de1e1386a2b328e3/lib/src/Puzzle.dart#L47) ends up being redundant, we can replace with:

```suggestion
   Duration getTimeElapsed() => _stopwatch.elapsed;
```

Reply:
Check the comment above about the [Line 47](https://github.com/JoseGeorges8/sudoku-api/blob/95b4dff6533dbff802540617de1e1386a2b328e3/lib/src/Puzzle.dart#L47)